readthedocs/profiles/views.py

```diff
@@ -52,8 +53,29 @@
 from readthedocs.projects.utils import get_csv_file
 
 
+log = structlog.get_logger(__name__)
+
+
 class LoginViewBase(AllAuthLoginView):
-    pass
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+        last_login_method = self.request.COOKIES.get("last-login-method")
+        context_data["last_login_method"] = last_login_method
+
+        last_login_tab = None
```

Comment:
Should probably default to "vcs" just to be explicit. The front end could use a null value and the tab page focused in this case would be blank.